js/package.json

```diff
@@ -59,6 +60,9 @@
     "angular-sanitize": "^1.8.2",
     "debug": "^4.3.1",
     "masonry-layout": "^4.2.2",
-    "moment": "^2.29.1"
+    "moment": "^2.29.1",
+    "ngVue": "^1.7.8",
+    "vue": "^2.6.14",
+    "vue-loader": "^15.9.7"
```

Comment:
Isn't `vue-loader` more of a devDependency? Also the `vue-template-compiler` is missing.